src/app/components/Brand/index.stories.jsx

```diff
@@ -2,4 +2,5 @@ import React from 'react';
 import { storiesOf } from '@storybook/react'; // eslint-disable-line import/no-extraneous-dependencies
 import Brand from './index';
 
-storiesOf('Brand', module).add('default', () => <Brand />);
+storiesOf('Brand', module).add('default', () => <Brand indentedLogo={false} />);
```

Comment:
Would this not work if we did not pass in `indentedLogo` at all? Then the ternary in Brand could be as follows:

```
const StyledGridItem = styled.div`
    ${indentedLogo === true ? layoutGridItem : layoutGridItemFullWidth};
  `;
```

This way we would be utilising the default prop of false

Reply:
@pjlee11 I've addressed your comment by making the default prop value of `indentedLogo` as being `true`, since this is the version used in the header, the default usecase.